lib/commons/color/get-background-color.js

```diff
@@ -53,17 +53,17 @@ function getBgColor(elm, elmStyle) {
  * @param {Element} bgNode
  * @return {Boolean}
  */
-function contentOverlapping(targetElement, bgNode) {
+function contentOverlapping(targetElement, fgNode) {
 	// get content box of target element
-	// check to see if the current bgNode is overlapping
-	var targetRect = targetElement.getClientRects()[0];
```

Comment:
When the input was under audit and its parent was an implicit, inline label, the label would be skipped and not considered a background unless it was added to the stack manually. (also the label itself as a node, which contains text content). I couldn't guarantee which element was in the front since it was evaluating both the child and the parent for overlapping text content. If you look at the examples in this MDN doc, you can see `getClientRects` can be used to check inline content overlap. https://developer.mozilla.org/en-US/docs/Web/API/Element/getClientRects

But if this works, awesome. I want to run it through the aXe extension to be sure.